examples/README.md

```diff
@@ -14,15 +14,44 @@ and streaming APIs.
 You can find examples that demonstrate these APIs
 in the [`examples/src/`](./src) directory.
 
+* [How to use the one-step APIs](./src/onestep_defaults.py)
+* [How to change the algorithm suite](./src/onestep_unsigned.py)
+* [How to stream data in memory](./src/in_memory_streaming_defaults.py)
+* [How to stream data between files](./src/file_streaming_defaults.py)
+
 ## Configuration
 
 To use the library APIs,
 you need to describe how you want the library to protect your data keys.
 You can do this using
-[keyrings][#keyrings] or [cryptographic materials managers][#cryptographic-materials-managers],
-or using [master key providers][#master-key-providers].
+[keyrings](#keyrings) or [cryptographic materials managers](#cryptographic-materials-managers),
+or using [master key providers](#master-key-providers).
 These examples will show you how.
```

Comment:
Say something about how these are organized (e.g. KMS v. raw). Otherwise, the first thing people see is KMS and they might not understand why.